1_get_raw_data/src/get_raw_data.R

```diff
@@ -1,8 +1,10 @@
 library(readxl)
 
-get_sample <- function(file){
-  
+get_sample <- function(ind_file){
   
+  gd_get(ind_file, type='xlsx')
```

Comment:
looks good. i think specifying `type` is unnecessary when it's a non-Google file type like .xlsx, but there's also no harm as far as i know